src/nerve_tools/configure_nerve.py

```diff
@@ -75,6 +79,14 @@ def generate_subconfiguration(
     # hacheck will simply ignore the healthcheck_uri for TCP mode checks
     healthcheck_uri = service_info.get('healthcheck_uri', '/status')
     healthcheck_mode = service_info.get('healthcheck_mode', mode)
+    custom_labels = {}
+    try:
+        path = os.path.join(labels_dir, service_name + str(port) + '*')
+        for label_file in glob(path):
+            with open(label_file) as f:
+                custom_labels.update(yaml.load(f))
```

Comment:
This is how you can use safe_load with the CLoader
https://github.com/Yelp/service_configuration_lib/blob/master/service_configuration_lib/__init__.py#L26